src/js/index.js

```diff
@@ -0,0 +1,24 @@
+import '../css/style.css';
+import scoresList from './get-data.js';
+import postData from './post-data.js';
+
+require('@babel/polyfill');
+
+const leaderboard = document.querySelector('.leaderboard');
+const refresh = document.querySelector('#refresh');
+const form = document.querySelector('#form');
+const userInput = document.querySelector('#user');
+const scoreInput = document.querySelector('#score');
+const message = '<h6 class="text-center">Fetch Data by clicking refresh button</h6>';
+
+leaderboard.innerHTML = message;
+
+refresh.addEventListener('click', () => {
+  scoresList();
+  leaderboard.innerHTML = '';
+});
+
+form.addEventListener('submit', (e) => {
+  postData(userInput.value, scoreInput.value);
+  e.target.reset();
+});
```

Comment:
- [ ] Currently, the leaderboard emptied when I reload the page. Kindly make a line of code possibly a function call here to populate the leaderboard with API data whenever the page loads or reloads. :pray: 
- [ ] OPTIONAL: YOU may want to add validation to the form if `userInput.value` and `scoreInput.value` are valid inputs and display error message if not. This is just a stretch requirement and you can do it or leave it as you wish.